examples/oneapi/cpp/source/svm/svm_multi_class_thunder_csr_batch.cpp

```diff
@@ -0,0 +1,58 @@
+/*******************************************************************************
+* Copyright 2023 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include "oneapi/dal/algo/svm.hpp"
+#include "oneapi/dal/io/csv.hpp"
+
+#include "example_util/utils.hpp"
+
+namespace dal = oneapi::dal;
+
+int main(int argc, char const *argv[]) {
+    const auto train_data_file_name =
+        get_data_path("svm_multi_cls_synthetic_3k_train_sparse_data.csv");
```

Comment:
Please align data files naming with "dense" examples: 
```suggestion
        get_data_path("svm_multi_class_train_sparse_data.csv");
```